ntp-daemon/src/config/mod.rs

```diff
@@ -57,6 +57,60 @@ pub struct Config {
     #[cfg(feature = "sentry")]
     #[serde(default)]
     pub sentry: SentryConfig,
+    #[serde(default)]
+    pub observe: ObserveConfig,
+    #[serde(default)]
+    pub configure: ConfigureConfig,
+}
+
+fn default_observe_path() -> PathBuf {
+    PathBuf::from("/run/ntpd-rs/observe")
+}
+
+const fn default_observe_permissions() -> u32 {
+    0o777
+}
+
+#[derive(Clone, Deserialize, Debug)]
+pub struct ObserveConfig {
+    #[serde(default = "default_observe_path")]
+    pub path: PathBuf,
+    #[serde(default = "default_observe_permissions")]
+    pub mode: u32,
+}
+
+fn default_configure_path() -> PathBuf {
+    PathBuf::from("/run/ntpd-rs/configure")
+}
+
+const fn default_configure_permissions() -> u32 {
+    0o777
+}
```

Comment:
this is convenient for testing, but maybe not the best default in practice? thoughts?

Reply:
Small and need not be fixed before this gets merged, but I would change the configure default to limit that one to the user/group of the daemon.